nimble/host/src/ble_hs.c

```diff
@@ -699,7 +710,10 @@ ble_hs_init(void)
      * the application to configure this package in the meantime.
      */
 #ifdef MYNEWT
-    ble_npl_eventq_put((struct ble_npl_eventq *)os_eventq_dflt_get(), &ble_hs_ev_start);
+#if MYNEWT_VAL(BLE_HS_AUTO_START)
```

Comment:
I guess this `#if` should wrap complete `#if #else` instead of only mynewt-specific part since you also updated ports to work with this new setting

Reply:
Thanks, @andrzej-kaczmarek.  I have fixed it.